pkg/varlinkapi/containers.go

```diff
@@ -194,15 +195,25 @@ func (i *LibpodAPI) ListContainerChanges(call iopodman.VarlinkCall, name string)
 }
 
 // ExportContainer ...
-func (i *LibpodAPI) ExportContainer(call iopodman.VarlinkCall, name, path string) error {
+func (i *LibpodAPI) ExportContainer(call iopodman.VarlinkCall, name, outPath string) error {
 	ctr, err := i.Runtime.LookupContainer(name)
 	if err != nil {
 		return call.ReplyContainerNotFound(name)
 	}
-	if err := ctr.Export(path); err != nil {
+	outputFile, err := ioutil.TempFile("", "varlink_recv")
```

Comment:
This path should be "namespaced" like import to prevent two remote users stomping on each other.

Reply:
tempfile handles that